### ISSUE TITLE
Request: More detailed documentation

### ISSUE THREAD

**Issue by zhouzq-thu** [pytorch/pytorch#4424]
More detailed documentation such as

<img width="712" alt="screen shot 2018-01-01 at 12 28 29 pm" src="https://user-images.githubusercontent.com/9458157/34465601-59cd3d92-eeef-11e7-952d-057aa638b93c.png">

**Comment by apaszke** [comment]
@zhouzq-thu That looks awesome! Could you please submit a PR? We'll happily merge such improvements

**Comment by zhouzq-thu** [comment]
@apaszke I have submitted a PR (#4428), which contains the detailed documentation for some loss functions. Other function will be added later.

**Comment by apaszke** [comment]
That's some really good stuff. Doc improvements are always desired, but we simply don't have enough throughput to do all of this work ourselves. I'll close the issue, but if you'll find some time and improve other docstrings please send PRs!

**Comment by apaszke** [comment]
BTW what theme did you use to render that image?

**Comment by zhouzq-thu** [comment]
The original sphinx style.
